cms/topics/viewsets.py

```diff
@@ -0,0 +1,132 @@
+from typing import TYPE_CHECKING, ClassVar
+
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.ui.tables import Column, DateColumn
+from wagtail.admin.ui.tables.pages import PageStatusColumn
+from wagtail.admin.views.generic.chooser import ChooseResultsView, ChooseView
+from wagtail.admin.viewsets.chooser import ChooserViewSet
+from wagtail.coreutils import resolve_model_string
+
+from cms.articles.models import ArticleSeriesPage, StatisticalArticlePage
+from cms.methodology.models import MethodologyPage
+
+if TYPE_CHECKING:
+    from wagtail.models import Page
+    from wagtail.query import PageQuerySet
+
+
+class FeaturedSeriesPageMixin:
```

Comment:
Nit: Shouldn't this be in the `mixins.py` file. We also have another mixin below called `HighlightedChildPageMixin`.

We have other `mixins` files in the codebase, so thinking it might make sense to extract out the mixins into one mixin file.

Nothing pressing or major!

Reply:
this is more of contextul decision. You'd move them if they are more general purpose (i.e. to be used in other apps/modules) or if there is a bigger number of them

Leaving as is (but improved their name a tad in 708da0f)

Reply:
Thanks, Dan, for the class name improvements!